guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md

```diff
@@ -422,6 +426,8 @@ The following example removes the  `FL_ALLOWED_SEVERITIES` foreign key by deleti
     </table>
 </schema>
 ```
+{: .bs-callout .bs-callout-info}
```

Comment:
Sometime note gets squirrelly if there is no blank line above it. I know it affects headings, so just pointing this out in case of...

Reply:
Thanks @hguthrie!